src/map_extras.cpp

```diff
@@ -581,21 +581,11 @@ static bool mx_military( map &m, const tripoint & )
                     m.add_spawn( mon_dispatch, 1, *p );
                 }
             } else {
-                m.add_spawn( mon_zombie_soldier, 1, *p );
-                // 10% chance of zombie carrying weapon so 90% chance of it being on the ground
-                if( !one_in( 10 ) ) {
-                    item_group_id group;
-                    // 80% assault rifles, 10% LMGs, 5% shotguns, 5% sniper rifles
-                    if( one_in( 20 ) ) {
-                        group = Item_spawn_data_military_standard_sniper_rifles;
-                    } else if( one_in( 19 ) ) {
-                        group = Item_spawn_data_military_standard_shotguns;
-                    } else if( one_in( 9 ) ) {
-                        group = Item_spawn_data_military_standard_lmgs;
-                    } else {
-                        group = Item_spawn_data_military_standard_assault_rifles;
-                    }
-                    m.place_items( group, 100, *p, *p, true, calendar::start_of_cataclysm );
+                if ( one_in( 3 ) ) {
```

Comment:
Ick, I didn't realize this extra was still hardcoded.

Reply:
I don't know enough about things to un-hardcode this. If you want to do that, be my guest.